conduct.md

```diff
@@ -116,7 +116,7 @@ TODO: Write an introductory paragraph.
 
 ### Conflicts
 
-TODO: Write an introductory paragraph.
+When a conflict between students arise, it is imperative to deal with the situation in a respectful, yet effective way. Communication is a great tool to prevent or defuse conflicts between students, team members, or teams as a whole. According to Conway's Law the final product mirrors the methods of communication that went into the product's development. So if conflicts occur and prevent communication, it will likely lower the quality of the final product. For these reasons it is crucial to adhere to the codes of conduct relate to conflicts outlined below.
```

Comment:
1- Please split up this line to multiple lines with shorter length. This will avoid markdownlint error
2- `adhere to the codes of conduct relate to conflicts` I think there is a grammar issue with `relate` in this statement. It could be changed to `that relate` or maybe `relating`

Reply:
Makes sense. Thank you!